### DISCUSSION TITLE
Redirect cloaking_rules to a certain IP

### DISCUSSION THREAD

**Discussion by marcelloinfoweb** [DNSCrypt/dnscrypt-proxy#2697] [Q&A]
How to redirect cloaking_rules, for example restrict.youtube.com to a certain IP?
I would like to use it for my son's devices.

**Comment by lifenjoiner** [comment]
Is this what you are looking for? https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Cloaking
It is a further explanation of the example file.
